calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -412,6 +450,47 @@ public CalendarEventVector getEvents(List references, TimeRange range)
 		return this.getEvents(references, range, false);
 	}
 
+	public List<CalendarEvent> getFilteredEvents(Map<EventFilterKey, Object> options) {
+
+		if (options == null) options = Collections.emptyMap();
```

Comment:
I'd prefer this on different lines and using brackets.

Reply:
Just out of interest, why?

Reply:
Just good practices, I remember being asked that on previous PRs. I guess the reason was it's safer to do it as anyone coming into it in the future will already have them placed, in case they want to add more lines within the if.

But not a big deal, I saw at least another one but didn't comment it anyway.

Reply:
I agree about the brackets rule if you have the if and the block on two lines without a bracket. That's dangerous as it's not clear what block belongs to the if. I'll change it.

Reply:
I've made it a ternary. options can be null, that assignment of emptyMap is just to make the code further down clearer. I can't imaging somebody would want to add another statement to that block. No logging is needed as it's an expected condition. Maybe I should have used Optional, but I'd still have had to test that each time I accessed the map.

Reply:
got to say, I still prefer the original ...

Reply:
Yeah, stylistically I can understand. However, @bgarciaentornos brings up a good point that it can easily lead to errors if someone not familiar with the code makes the wrong modification. At least with the braces it's very clear what's inside the if block and what's not.

This has been a standard enforced in the Sakai code base for as long as I remember, and I think we should continue to apply these standards going forwards.